Sources/SKSupport/Sequence+AsyncMap.swift

```diff
@@ -12,7 +12,7 @@
 
 extension Sequence {
   /// Just like `Sequence.map` but allows an `async` transform function.
-  func asyncMap<T>(
+  public func asyncMap<T>(
```

Comment:
Do these need to be public or can they be package (or SPI)? Don't love adding public extensions onto `Sequence`.

Reply:
Adopting `package` in SourceKit-LSP would be good in general. We currently still use `public` for anything that’s  used cross-module. I filed https://github.com/apple/sourcekit-lsp/issues/1315 to adopt `package` in general.